store/sdk/src/test/java/org/apache/carbondata/sdk/file/ImageTest.java

```diff
@@ -1168,4 +1169,44 @@ public void writeCarbonFile(String path, int num) throws Exception {
     reader.close();
   }
 
+  @Test public void testHugeBinaryWithComplexType()
```

Comment:
Can you also add a testcase to read the written files by CarbonSession?

Reply:
yes, a testcase with sql statement using carbonsession needed

Reply:
ok. Added now